zetaclient/bitcoin/fee.go

```diff
@@ -0,0 +1,234 @@
+package bitcoin
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/common/bitcoin"
+	clientcommon "github.com/zeta-chain/zetacore/zetaclient/common"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/pkg/errors"
+)
+
+const (
+	bytesPerKB              = 1000
+	bytesEmptyTx            = 10  // an empty tx is 10 bytes
```

Comment:
"bytesEmptyTx", Seems like this isn't used, will it be used in the future?

Reply:
Yeah. I just noticed and removed it.